cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -151,6 +151,18 @@ write_files:
         done;
         export HOME="${!HOME_BAK}"
       }
+
+      function publish_startup_time
+      {
+        StartTime=$1
+        EndTime=$2
+        METRIC_NAMESPACE="ParallelCluster"
```

Comment:
I'd avoid creating `METRIC_NAMESPACE` and `METRIC_NAME` since they are used in a single place.

Reply:
Done